examples/2d/contributors.rs

```diff
@@ -52,19 +60,328 @@ const ALPHA: f32 = 0.92;
+    "Mariusz Kryński",
+    "Mark",
+    "Martin Lavoie",
+    "Martín Maita",
+    "Martin Svanberg",
+    "Mat Hostetter",
+    "Matteo Guglielmetti",
+    "Matthias Seiffert",
+    "Max Bruckner",
+    "maxwellodri",
+    "memoryruins",
+    "mfrancis107",
+    "MGlolenstine",
+    "Michael Hills",
+    "Michael Tang",
+    "Mika",
+    "Mikail Khan",
+    "Mike",
+    "Milan Vaško",
+    "milkybit",
+    "MinerSebas",
+    "Minghao Liu",
+    "MiniaczQ",
+    "Mirko Rainer",
+    "Moxinilian",
+    "MsK`",
+    "multun",
+    "Nathan Jeffords",
+    "Nathan Stocks",
+    "Nathan Ward",
+    "Nibor62",
+    "Nicholas Rishel",
+    "Nick",
+    "Nikita Zdanovitch",
+    "Niklas Eicker",
+    "Noah Callaway",
+    "Nolan Darilek",
+    "Olivier Pinon",
+    "OptimisticPeach",
+    "Oscar",
+    "Patrick Greene",
+    "Patrik Buhring",
+    "Paweł Grabarz",
+    "Philip Degarmo",
+    "Philipp Mildenberger",
+    "Piotr Balcer",
+    "Plecra",
+    "Protowalker",
+    "Psychoticpotato",
+    "r00ster",
+    "Raymond",
+    "RedlineTriad",
+    "reidbhuntley",
+    "Rémi Lauzier",
+    "Renato Caldas",
+    "Restioson",
+    "Richard Tjerngren",
+    "RiskLove",
+    "rmsthebest",
+    "Rob",
+    "Robbie Davenport",
+    "Robert Swain",
+    "Rob Parrett",
+    "rod-salazar",
+    "Ryan Lee",
+    "Ryan Scheel",
+    "sapir",
+    "sark",
+    "Saverio Miroddi",
+    "Schell Carl Scivally",
+    "sdfgeoff",
+    "Sergey Minakov",
+    "simens_green",
+    "simlay",
+    "Simon Guillot",
+    "Smite Rust",
+    "speak",
+    "Spencer Burris",
+    "Squirrel",
+    "StarArawn",
+    "stefee",
+    "Stjepan Glavina",
+    "SvenTS",
+    "szunami",
+    "taryn",
+    "TehPers",
+    "Telzhaak",
+    "TEMHOTAOKEAHA",
+    "terrarier2111",
+    "thebluefish",
+    "Theia Vogel",
+    "the-notable",
+    "Théo Degioanni",
+    "TheRawMeatball",
+    "therealstork",
+    "Thirds",
+    "Thomas Heartman",
+    "Thomas Herzog",
+    "Tiago Ferreira",
+    "tiagolam",
+    "tigregalis",
+    "Tomasz Sterna",
+    "Tom Bebb",
+    "Toniman20",
+    "Toothbrush",
+    "TotalKrill",
+    "Tristan Pemble",
+    "Utkarsh",
+    "Valentin",
+    "verzuz",
+    "Victor \"multun\" Collod",
+    "VitalyR",
+    "Vladyslav Batyrenko",
+    "VVishion",
+    "walterpie",
+    "Waridley",
+    "W. Brian Gourlie",
+    "Will Crichton",
+    "Will Dixon",
+    "Will Hart",
+    "William Batista",
+    "willolisp",
+    "Wojciech Olejnik",
+    "Wouter Buckens",
+    "Wouter Standaert",
+    "wyhaya",
+    "Xavientois",
+    "Yoh Deadfall",
+    "Zach Gotsch",
+    "Zaszi",
+    "Zhixing Zhang",
+    "Zicklag",
+    "Zooce",
+];
+
+fn setup_contributor_selection(
     mut commands: Commands,
     asset_server: Res<AssetServer>,
     mut materials: ResMut<Assets<ColorMaterial>>,
 ) {
-    let contribs = contributors();
+    let contribs = contributors().unwrap_or_else(|_| {
+        BTreeSet::from_iter(CONTRIBUTORS_LIST.iter().map(|name| name.to_string()))
```

Comment:
Ooh fancy. If we're going to use a relatively advanced exotic data structure in a basic example, we should have a basic comment justifying its choice to help teach beginners.

Reply:
Added comment to these lines